packages/common/src/types/field.ts

```diff
@@ -264,6 +264,7 @@ export enum MetricType {
     MEDIAN = 'median',
     STRING = 'string',
     DATE = 'date',
+    TIMESTAMP = 'timestamp',
```

Comment:
We need to make sure Timestamp is handled everywhere in the codebase. Search the repo by `MetricType.DATE`.

eg: we need timestamp in this list
```
const NonAggregateMetricTypes = [
    MetricType.STRING,
    MetricType.NUMBER,
    MetricType.DATE,
    MetricType.BOOLEAN,
];
```